pgx-tests/src/tests/array_tests.rs

```diff
@@ -255,6 +302,51 @@ mod tests {
         assert_eq!(json.0, json! {{"values": [1, 2, 3, null, 4]}});
     }
 
+    #[pg_test]
+    fn test_arr_data_ptr() {
+        let len = Spi::get_one::<i32>("SELECT get_arr_nelems('{1,2,3,4,5}'::int[])")
+            .expect("failed to get SPI result");
+
+        assert_eq!(len, 5);
+    }
+
+    #[pg_test]
+    fn test_get_arr_data_ptr_nth_elem() {
+        let nth = Spi::get_one::<i32>("SELECT get_arr_data_ptr_nth_elem('{1,2,3,4,5}'::int[], 2)")
+            .expect("failed to get SPI result");
+
+        assert_eq!(nth, 3);
+    }
+
+    #[pg_test]
+    fn test_display_get_arr_nullbitmap() {
+        let bitmap_str = Spi::get_one::<String>(
+            "SELECT display_get_arr_nullbitmap(ARRAY[1,NULL,3,NULL,5]::int[])",
+        )
+        .expect("failed to get SPI result");
+
+        assert_eq!(bitmap_str, "0b00010101");
```

Comment:
Is this to show which indexes have nulls and which indexes have non-null (apologies for the crude ascii diagram that may not render correctly)?

```
0b00010101
    [5n3n1]
```

Reply:
Correct. @mhov wrote that test, but it basically prints the bitwise repr of the null bitmap that Postgres uses internally.

Reply:
I believe it's actually the reverse though:
```
00001010
   5n3n1
```

Reply:
This tendency in the formatting detail needs to be documented.

Reply:
What were you saying, @workingjubilee? It is what Brady just said, as witnessed here in this very test. The nullbitmap uses a 0 to mark "is null", unlike all the rest of Postgres and `NullableDatum`, which uses `true` bools to mark "is null", because Postgres has very good ideas.